config/crd/bases/elbv2.k8s.aws_targetgroupbindings.yaml

```diff
@@ -40,143 +40,262 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
-  validation:
```

Comment:
This looks to the old format. Do I have a different version of `controller-gen`? I use 0.4.0 so I think this is fine to check in.

Reply:
the controller-gen seems to have upgraded, i haven't updated the CRDs since launch, need to figure out what's changed

Reply:
Did you get a chance to take a look. Let me know if you want me to add any changes.

Reply:
it's fine to include these CRD changes. we'll adjust it if needed when cutting a release